docs/part3/commonstatsmethods.md

```diff
@@ -1015,11 +1007,15 @@ Imposing physical boundaries (such as requiring $\mu>0$ for a signal strength) i
 
 The boundary is imposed by **restricting the parameter range(s)** to those set by the user, in the fits. Note that this is a trick! The actual fitted value, as one of an ensemble of outcomes, can fall outside of the allowed region, while the boundary should be imposed on the physical parameter. The effect of restricting the parameter value in the fit is such that the test statistic is modified as follows ;
 
-$q(x) = - 2 \ln \mathcal{L}(x,\hat{\hat{\theta}}(x))/\mathcal{L}(\hat{x},\hat{\nu})$, if $\hat{x}$ in contained in the bounded range
+$$q(x) = - 2 \ln \mathcal{L}(x,\hat{\hat{\theta}}(x))/\mathcal{L}(\hat{x},\hat{\nu}),$$
```

Comment:
I didn't notice this before, but I think $\theta$ should be $\nu$ here, right? And While we're at it, perhaps we should also change this notation to be more consistent with the paper? i.e. using $\mu$ where currently $x$ is being used? (Here and on a few other lines in this section)

Reply:
You're right, should be \nu instead of \theta and we can use \vec{\mu} as elsewhere instead of "x" - can you make these changes?